docs/api-reference/globe-viewport.md

```diff
@@ -0,0 +1,99 @@
+# GlobeViewport Class (Experimental)
+
+The `GlobeViewport` class takes globe view states (`latitude`, `longitude`, and `zoom`), and performs projections between world and screen coordinates. It is a helper class for visualizing the earth as a 3D globe.
```

Comment:
My general direction has been to move away from named viewport subclasses and have the views just populate a Viewport and return it, so we don't need to document one more class for every view.

Maybe that is not possible in this case.

Reply:
This viewport has to implement quite a few overrides, including project/unproject and getBounds. The base Viewport class is already bloated trying to handle both geospatial and non-geospatial use cases, with WebMercator assumptions baked in. It will get harder and harder as we start generalizing the projection system.